recipes/libsass/all/conanfile.py

```diff
@@ -43,21 +58,64 @@ def _configure_autotools(self):
         self._autotools.configure(args=args)
         return self._autotools
 
-    def build(self):
+    def _build_autotools(self):
         with tools.chdir(self._source_subfolder):
             tools.save(path="VERSION", content="%s" % self.version)
+            self.run("{} -fiv".format(tools.get_env("AUTORECONF")))
             autotools = self._configure_autotools()
             autotools.make()
 
-    def package(self):
+    @property
+    def _make_program(self):
+        return tools.get_env("CONAN_MAKE_PROGRAM", tools.which("make") or tools.which("mingw32-make"))
+
+    def _build_mingw(self):
+        makefile = os.path.join(self._source_subfolder, "Makefile")
+        tools.replace_in_file(makefile, "CFLAGS   += -O2", "")
+        tools.replace_in_file(makefile, "CXXFLAGS += -O2", "")
+        tools.replace_in_file(makefile, "LDFLAGS  += -O2", "")
+        with tools.chdir(self._source_subfolder):
+            env_vars = AutoToolsBuildEnvironment(self).vars
+            env_vars.update({
+                "BUILD": "shared" if self.options.shared else "static",
+                "PREFIX": tools.unix_path(os.path.join(self.package_folder)),
+                # Don't force static link to mingw libs, leave this decision to consumer (through LDFLAGS in env)
+                "STATIC_ALL": "0",
+                "STATIC_LIBGCC": "0",
+                "STATIC_LIBSTDCPP": "0",
+            })
+            with tools.environment_append(env_vars):
+                self.run("{} -f Makefile".format(self._make_program))
+
+    def build(self):
+        if self._is_mingw:
+            self._build_mingw()
+        else:
+            self._build_autotools()
+
+    def _install_autotools(self):
         with tools.chdir(self._source_subfolder):
             autotools = self._configure_autotools()
             autotools.install()
-        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
         tools.remove_files_by_mask(self.package_folder, "*.la")
 
+    def _install_mingw(self):
+        self.copy("*.h", dst="include", src=os.path.join(self._source_subfolder, "include"))
```

Comment:
so, `make install` doesn't work for MinGW?

Reply:
Generally speaking, it should work, but install target calls something that didn't work on my computer. Moreover it installs runtime in lib folder, so it's just easier to manually copy these files.